src/probabilities_estimators/histograms/rectangular_binning.jl

```diff
@@ -20,14 +21,13 @@ struct RectangularBinning{E} <: AbstractBinning
 end
 
 function probabilities(x::Vector_or_Dataset, binning::RectangularBinning)
-    fasthist(x, binning)[1]
+    fasthist!(x, binning)[1]
 end
+
 function probabilities(x::Vector_or_Dataset, ε::Union{Real, Vector{<:Real}})
     probabilities(x, RectangularBinning(ε))
 end
 
-
-
 """
     RectangularBinEncoder(x, binning::RectangularBinning) <: SymbolizationScheme
```

Comment:
Since this is now a symbolization scheme, the strictly-symbolization-related code should be move to a separate file in the `symbolization` folder, just like `GaussianSymbolization.jl` and `OrdinalPattern.jl`.

Reply:
Well, I disagree here because this file is the most intuitive file to leave this function at. Similarly, we don't move the `probabilities_and_events` into a different folder. If I'm going through the code that does the rectangular binning, I'd much rather have this all in the same place. Mapping into bins is kind of the whole idea behind the histogram of bins.

Reply:
Sure, this is by no means critical. Just keep it as if you prefer!